index.html

```diff
@@ -1,40 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link rel="stylesheet" href="./Styles/styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
-    />
-  </head>
-  <body>
-    <header>
-      <a href="index.html" class="logo"><h1>Byiringiro Moise</h1></a>
-      <span class="material-symbols-outlined"> menu </span>
-    </header>
-    <section class="profile">
-        <div class="introduction">
-<h2>Hey There. I'm Moise</h2>
-<h3>I'am a Software Developer</h3>
-<p>
-    I can help you build a product, feature or website look through some of my work and experience!
-    if you like what you see and have a project you need coded, don't hesitate to contact me.
-</p>
-        </div>
-        <div>
-          <ul class="social-media-icons">
-<li><a href=""><img src="/items/Github-icon.png" alt="github icon"></a></li>
-<li><a href=""><img src="/items/linkedin-icon.png" alt=""></a></li>
-<li><a href=""><img src="/items/Icon - Angelist.png" alt=""></a></li>
-<li><a href=""><img src="/items/Icon - Twitter.png" alt=""></a></li>
-<li><a href=""><img src="/items/M-icon.png" alt=""></a></li>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="./Styles/styles.css" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+
+<body>
+  <header>
+    <a href="index.html" class="logo">
+      <h1>Byiringiro Moise</h1>
+    </a>
+    <span class="material-symbols-outlined"> menu </span>
+  </header>
+  <section class="profile">
+    <div class="introduction">
+      <h2>Hey There. I'm Moise</h2>
+      <h3>I'am a Software Developer</h3>
```

Comment:
![Screenshot from 2022-09-23 15-33-15](https://user-images.githubusercontent.com/93039359/191943079-2317f9d8-c42c-4ea3-9e5d-def3a5fd66ff.png)


- [ ] You have tried to solve the overflow issue but the previous reviewer suggested you to remove overflow so you won't see any whitespace on the right side of your page. I can see that kindly fix this issue by giving widths in percentage instead of pixels. :blush:

Reply:
hi there, I have fixed it by giving the body a width of 100% and it is working perfectly in my computer there are no scrolls on the left and right.

Reply:
Hi Moise-